src/main/java/io/phdata/streamliner/schemadefiner/mapper/SnowflakeMapper.java

```diff
@@ -96,25 +123,37 @@ private static SnowflakeTable mapSchemaCrawlerTable(Table table, Snowflake desti
 
     return new SnowflakeTable(
         table.getName(),
-        getSnowflakeTableName(table.getName(), destination),
+        getSnowflakeTableName(table.getName()),
         table.getRemarks(),
         primaryKey,
         null,
         null,
         null,
-        new FileFormat(table.getName(), "PARQUET"),
+        createFileFormat(table.getName()),
         mapSchemaCrawlerColumnDefinition(columns));
   }
 
-  public static String getSnowflakeTableName(String tableName, Snowflake destination) {
+  private FileFormat createFileFormat(String tableName) {
+    if (attributeRetriever == null) {
```

Comment:
When is this condition met and how can we assume the filetype is Parquet?

Reply:
I will be changing the default condition in `AttributeRetrieverFactory`. Based on that this will get changed. By default we provide fileType as `Parquet` . This can be overridden using `userDefinedTables` param under source